tests/unit/App.spec.ts

```diff
@@ -6,6 +6,7 @@ import { createPinia, setActivePinia } from 'pinia'
 import { useStore } from '@/store/store'
 import { shallowMount, createLocalVue } from '@vue/test-utils'
 import { AxiosInstance as axios } from '@/utils'
+import { shallowWrapperFactory } from '../jest-wrapper-factory'
```

Comment:
I wonder if we should change the file's name here. Something like: `vitest-wrapper-factory` 😆 .

Reply:
Good point. I'll do it.

Reply:
Thanks Sev! Resolving.